README.md

```diff
@@ -0,0 +1,146 @@
+# Capsule sandbox
+
+The Capsule sandbox is a containerized development environment for building Capsule applications in Rust.
+
+## Table of Contents
+
+* [Introduction](#introduction)
+* [Getting Started](#getting-started)
+  * [Vagrant and VirtualBox](#vagrant-and-virtualbox)
+  * [Linux Distributions](#linux-distributions)
+  * [Without Docker](#without-docker)
+  * [Kernel NIC Interface](#kernel-nic-interface)
+* [Packaging for Release](#packaging-for-release)
+* [Contributing](#contributing)
+* [Code of Conduct](#code-of-conduct)
+* [License](#license)
+
+## Introduction
+
+`Capsule` is built on Intel's [Data Plane Development Kit](https://www.dpdk.org/) release 18.11. While it is written in Rust, it needs to be able to call `DPDK`'s C functions installed separately on the host as shared dynamic libaries. To make developing and running Capsule applications easy, we created the sandbox container with all the necessary dependencies. You can pull down the container and start writing network functions in stable `Rust` right away.
+
+## Getting Started
+
+To run the sandbox, the docker host must run a Linux distribution. `DPDK` requires either Linux or FreeBSD. We may add FreeBSD support in the future.
+
+### Vagrant and VirtualBox
+
+The quickest way to start the sandbox is to use the Vagrant virtual machine included in this repository. If you are developing on either MacOS or Windows, this is the only way to write a `Capsule` application.
+
+After cloning this repository, you can start the vagrant VM using commands,
+
+```
+host$ vagrant up docker
```

Comment:
so, the default `vagrant up` will use the docker variation. Not sure you want to use that / call that out here.

Reply:
seems too much effort explaining vagrant without much gained

Reply:
yeah. maybe just `vagrant up` and not worry about it since you're talking containers?